package.json

```diff
@@ -111,7 +111,7 @@
     "react-redux": "^5.0.7",
     "react-router-dom": "^4.3.1",
     "react-router-prop-types": "^1.0.4",
-    "react-select": "^2.1.0",
+    "react-select": "^2.1.2",
```

Comment:
react-select? 
What are the pros/cons of building the select feature with HTML/CSS as compared to using a react library? @emmaadesile

Reply:
Using react-select makes the select feature less verbose and it handles the CSS.